dev/re_frisk/demo.cljs

```diff
@@ -241,7 +241,8 @@
                            "dispatch 3 events doing nothing"]]))}))
 
 (defn mount []
-  (rdom/render [simple-example] (js/document.getElementById "app"))
+  (let [root (rdclient/create-root (.getElementById js/document "app"))]
```

Comment:
could you please elaborate on this change, just wondering how it worked before

Reply:
do you know starting which version of React this will work?

Reply:
Apparently they introduce it in React 18, which corresponds with [reagent 1.2.0](https://github.com/reagent-project/reagent/blob/master/CHANGELOG.md#120-2023-02-16)

Reply:
It should work just the same. It's the change suggested by the React docs when you press in the warning link.